### DISCUSSION TITLE
fpm on Arch User Repository

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/fpm#467] [General]
*fpm* has recently entered the Arch User Repository (AUR) as [*fortran-fpm*](https://repology.org/project/fortran-fpm/versions). 

While certainly redundant naming, I can see reasons why [just *fpm* wouldn't work here](https://aur.archlinux.org/packages.php?K=fpm). I submitted it as just *fpm* to conda-forge, because ruby gems or php are not really strong on conda-forge and therefore [fpm was still free](https://anaconda.org/search?q=fpm).

Is there any naming guideline we want to establish for packaging *fpm* in case we run into conflicts with other similarly named projects?

**Comment by LKedward** [comment]
Perhaps using `'fortran-package-manager'` everywhere is best to avoid both ambiguity and redundancy?

**Reply by awvwgk** [reply]
We just have to make sure it can be found if the users are searching for fpm in their package manager (adding fpm as keyword or in the package description).
